ReactCommon/react/renderer/components/textinput/iostextinput/conversions.h

```diff
@@ -126,6 +126,30 @@ inline void fromRawValue(
   abort();
 }
 
+inline void fromRawValue(
+    const PropsParserContext &context,
+    const RawValue &value,
+    ReturnKeyAction &result) {
+  auto string = (std::string)value;
+  if (string == "default") {
```

Comment:
Where do we set `"default"`? Is this set somewhere when the component doesn't pass the prop?

Reply:
No, it should always be set to one of `"blur"`, `"blurAndSubmit"` or `"newline"`. This is removed now